services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -213,6 +213,11 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
         description="resource usage tracker service client's plugin",
     )
 
+    DIRECTOR_V2_PUBLIC_API_BASE_URL: AnyUrl = Field(
```

Comment:
question: any reason why you chose AnyUrl instead of AnyHttpUrl type?

Reply:
you are absolutely right.